types/conversations.d.ts

```diff
@@ -0,0 +1,159 @@
+import { Language } from 'parsimmon';
```

Comment:
this dependency is not in `package.json`

Reply:
This didn't even be imported here...